src/Database/Attach/File.php

```diff
@@ -866,14 +873,13 @@ protected function deleteEmptyDirectory($dir = null)
 
     /**
      * Returns true if a directory contains no files.
-     * @return void
+     *
+     * @param string $dir The path to the directory.
+     *
+     * @return bool|null
      */
     protected function isDirectoryEmpty($dir)
     {
-        if (!$dir) {
-            return null;
-        }
-
```

Comment:
What was the reason for this change? It conflicts with the docblock return type now

Reply:
You're right. Fixed the docblock.